bin/genesis

```diff
@@ -3567,7 +3677,11 @@ sub kit_file {
 		$dir = $KIT_CACHE_DIR{"$kit/$version"};
 	} else {
 		$dir = workdir('kit');
-		qx(tar -C $dir -xz --strip-components 1 -f .genesis/kits/$kit-$version.tar.gz $kit-$version/$relpath 2>/dev/null);
+		Genesis::ShellExec::check(
```

Comment:
Do an execute() here, since a check() makes it seem like we should be reacting.

Otherwise, let's react to the failure?